sdk/log/doc.go

```diff
@@ -1,9 +1,36 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-// TODO (#5065): Expand documentation stub.
-
 /*
 Package log provides the OpenTelemetry Logs SDK.
+
+See https://opentelemetry.io/docs/concepts/signals/logs/ for information
+about the concept of OpenTelemetry Logs and
+https://opentelemetry.io/docs/concepts/components/ for more information
+about OpenTelemetry SDKs.
+
+The entry point for the log package is [NewLoggerProvider].
+[LoggerProvider] is the object that all Bridge API calls use to create
+Loggers, and ultimately emit log records.
+Also, it is an object that should be used to
+control the life-cycle (start, flush, and shutdown) of the Logs SDK.
+
+A LoggerProvider needs to be configured to process the log records, this is
+done by configuring it with a [Processor] implementation using [WithProcessor].
+The log package provides [BatchProcessor] and [SimpleProcessor]
+that are configured with an [Exporter] implementation and
+exports the log records to given destination. See
+[go.opentelemetry.io/otel/exporters] for exporters that can be used as
```

Comment:
What exporters can be used as a processor?

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/5609/commits/f61e71ab43d90ccffb993ee56bdb3cc90ff76943